tests/covalent_dispatcher_tests/_cli/service_test.py

```diff
@@ -220,7 +220,7 @@ def test_start(mocker, monkeypatch, is_migration_pending, ignore_migrations):
 
     if ignore_migrations or not is_migration_pending:
         graceful_start_mock.assert_called_once()
-        set_config_mock.assert_called_once()
+        set_config_mock.assert_called()
```

Comment:
```suggestion
        assert set_config.call_count == 2
```
this might be a more stricter check to assert the exact number of times set_config is called within the `service.py`